src/index.html

```diff
@@ -1,27 +1,446 @@
+          class="card__photo"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <span class="card__product-code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <ul class="stars stars--4 card__stars">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div
+        data-qa="card"
+        class="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple Monoblock"
+          class="card__photo"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <span class="card__product-code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <ul class="stars stars--4 card__stars">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
+        >
+          Buy
+        </a>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="Apple Monoblock"
+          class="card__photo"
+        />
+
+        <h3 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <span class="card__product-code">Product code: 195434</span>
+
+        <div class="card__rating">
+          <ul class="stars stars--4 card__stars">
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+            <li class="stars__star"></li>
+          </ul>
+
+          <span class="card__reviews">Reviews: 5</span>
+        </div>
+
+        <div class="card__price-container">
+          <span class="card__price-label">Price:</span>
+
+          <span class="card__price-value">$2,199</span>
+        </div>
+
+        <a
+          href="#"
+          class="card__buy-link"
```

Comment:
it's not a link, it's a button)
```suggestion
          class="card__buy-button"
```

Reply:
@VolodymyrKirichenko I used "card__buy-button" before
but Kyrylo said, if we use tag `<a></a>` we need to name class with "link" and I renamed it

Reply:
> @VolodymyrKirichenko I used "card__buy-button" before but Kyrylo said, if we use tag `<a></a>` we need to name the class with "link" and I renamed it

@Valeraom  
`card__buy-link` - how to understand that this is a button? If I hadn't seen the demo, I would have thought it was just a link

Use `card__button` to explicitly indicate that the `<a>` tag functions as a button, even if it has a different tag.
it will be more clear. You'll always know it's a button.

Reply:
@VolodymyrKirichenko ok, probably I misunderstood this, because Kyrylo used your variant in today's meeting)